prow/plugins/lgtm/lgtm.go

```diff
@@ -83,6 +84,18 @@ func optionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Lg
 	return &plugins.Lgtm{}
 }
 
+// triggerOptionsForRepo gets the plugins.Trigger struct that is applicable to the indicated repo.
+func triggerOptionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Trigger {
```

Comment:
Any reason not to reuse `config.PluginConfig.TriggerFor()`?
This logic isn't specific to the `lgtm` plugin.